### ISSUE TITLE
[Desktop] Support for `SystemBackdrop`

### ISSUE THREAD

**Issue by xperiandri** [unoplatform/uno#17275]
### What would you like to be added

`Microsoft.UI.Composition.SystemBackdrops.DesktopAcrylicController` and `Microsoft.UI.Composition.SystemBackdrops.MicaController` to be implemented for platforms that support windows and support backdrop

### Why is this needed

To make apps beautiful

**Comment by mcNets** [comment]
I'm sorry to ask, but what do you mean exactly?

You can use Mica with the Windows platform.

**Comment by xperiandri** [comment]
@mcNets unless you put `#if WINDOWS` it crashes
``` C#
#if WINDOWS
if (Microsoft.UI.Composition.SystemBackdrops.DesktopAcrylicController.IsSupported())
{
    MainWindow.SystemBackdrop = new Microsoft.UI.Xaml.Media.DesktopAcrylicBackdrop();
}
#endif

**Comment by xperiandri** [comment]
Why not just return `false` in Uno?

**Comment by jeromelaban** [comment]
Sure, returning false makes sense here. The current code is auto generated, and that's its default behavior.

**Comment by xperiandri** [comment]
@MartinZikmund I remember you implemented this, right?

**Comment by MartinZikmund** [comment]
@xperiandri yes, the check no longer throws, but the `SystemBackdrop` itself is not supported yet

**Comment by MartinZikmund** [comment]
This is supported on macOS now, Win32 support is in progress by @morning4coffe-dev
